src/olympia/addons/serializers.py

```diff
@@ -276,8 +298,8 @@ def fake_object(self, data):
         self._attach_fields(
             obj, data,
             ('average_daily_users', 'bayesian_rating', 'created',
-             'default_locale', 'guid', 'hotness', 'icon_type',
-             'is_experimental', 'is_listed',
+             'default_locale', 'guid', 'has_eula', 'has_privacy_policy',
```

Comment:
I agree about "hotness" but I don't like changing model columns unless I absolutely have to. It's not exposed to the outside world, so at least the weirdness is strictly internal.